src/test/java/seedu/address/model/person/PhoneTest.java

```diff
@@ -13,20 +13,16 @@ public void constructor_null_throwsNullPointerException() {
         assertThrows(NullPointerException.class, () -> new Phone(null));
     }
 
-    @Test
-    public void constructor_invalidPhone_throwsIllegalArgumentException() {
-        String invalidPhone = "";
-        assertThrows(IllegalArgumentException.class, () -> new Phone(invalidPhone));
-    }
-
     @Test
     public void isValidPhone() {
         // null phone number
         assertThrows(NullPointerException.class, () -> Phone.isValidPhone(null));
 
+        // phone numbers now option, so can be empty strings
```

Comment:
Should this be "now optional"?

Reply:
Also the same comment as for the email - would it be better to just put "phone numbers optional" assuming others looking at the code in the future won't know phone numbers were initially compulsory?

Reply:
Agree with @tingxuanp about discarding the "now"